### PR TITLE
feat: Add Kinesis Labs

### PR DESCRIPTION
**Pull request by kinesis-labs**
## Description

Initial Commit for Kinesis Labs.

## Checklist

- [ ] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] (optional) As a contributor, my Ethereum address/ENS is:
- [ ] (optional) As a contributor, my Twitter handle is:

## How to test?

`pnpm install`
`pnpm dev`
[http://localhost:5001/apps/kinesis-labs/tokens?groupIds[]=pool&network=evmos](http://localhost:5001/apps/kinesis-labs/tokens?groupIds[]=pool&network=evmos)

### CODE REVIEW

File: src/apps/kinesis-labs/evmos/kinesis-labs.pool.token-fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @Inject(KinesisLabsOnChainReserveStrategy)
+    private readonly kinesisLabsOnChainReserveStrategy: KinesisLabsOnChainReserveStrategy,
+    @Inject(CurveVirtualPriceStrategy)
+    private readonly curveVirtualPriceStrategy: CurveVirtualPriceStrategy,
+    @Inject(KinesisLabsContractFactory) private readonly kinesisLabsContractFactory: KinesisLabsContractFactory,
+  ) {}
+
+  async getPositions() {
+    const basePools = await this.curvePoolTokenHelper.getTokens<KinesisLabsPool, Erc20>({
+      network: Network.ETHEREUM_MAINNET,
```
Comment (immasandwich) [inline src/apps/kinesis-labs/evmos/kinesis-labs.pool.token-fetcher.ts:39]: Should be `Network.EVMOS_MAINNET`

File: src/apps/kinesis-labs/evmos/kinesis-labs.pool.definitions.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+export const KINESIS_LABS_BASEPOOL_DEFINITIONS: CurvePoolDefinition[] = [
+  // Nomad Base Pool madUSDC/madUSDT/FRAX
+  {
+    swapAddress: '0x49b97224655AaD13832296b8f6185231AFB8aaCc',
+    tokenAddress: '0xfB25679ff0651cde4Cf58887be266B68326dDaB6',
+  },
+  // Celer Base Pool celerUSDC/celerUSDT/FRAX
+  {
+    swapAddress: '0xbBD5a7AE45a484BD8dAbdfeeeb33E4b859D2c95C',
+    tokenAddress: '0xF6b65B88A9e7846ED0DE503E682cC230F892C2fa',
```
Comment (immasandwich) [inline src/apps/kinesis-labs/evmos/kinesis-labs.pool.definitions.ts:12]: Lowercase all addresses 🙏

**Review by immasandwich — APPROVED** [review]

**Review by immasandwich — APPROVED** [review]

**Comment by SamIam-0x** [comment]
Hello @kinesis-labs !

Looks like you did not provide an address when submitting your PR. I recognize it may have seemed kind of random that we were asking for a wallet address with a PR submission, but we are actually looking for ways to reward Studio contributors in the future. 

Is there an address you’d want to provide? Could even be an empty address, if you do not have an doxxed address

Let me know! Also, if you’d like to privately provide it, feel free to shoot me a TG https://t.me/SamIAm_0x or DM on the Zapper Discord.

Thanks!
Sam

**Comment by SamIam-0x** [comment]
Gentle ping on the above @kinesis-labs ! Let me know

**Comment by kinesis-labs** [comment]
@SamIam-0x 
0x70b5d93d519C52a00839fEd5f87947D4Bc82ef6d

**Comment by SamIam-0x** [comment]
> @SamIam-0x 0x70b5d93d519C52a00839fEd5f87947D4Bc82ef6d

perfect, thanks!

**Pull request merged**